### PR TITLE
Fix for issue #86

### PR DESCRIPTION
**Pull request by ernado**
Quick fix for leading slashes on Windows.
Not sure about coverage of probable security issues on that platform.

Also moved client tests that only work on unix to separate file with build tags, tests are passing now on Windows.

For further discussion feel free to contact me at golang-ru (@ernado)

### CODE REVIEW

**Comment by ernado** [comment]
Switching to ubuntu for fixing unix part.

**Comment by kataras** [comment]
This was my previous atemp to fix it https://github.com/valyala/fasthttp/pull/85 but with domains, it also has problems with it but I closed my pull request because it wasn't good for all cases, you did it better nice!

**Comment by ernado** [comment]
@kataras 
My commit don't change existing logic for non-windows case, so this PR probably don't fix your issue.

Can you give more details on your issue with subdomains? I.e. what you are doing, what you expect and got instead. 
Thanks :)

File: client_unix_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
```
Comment (valyala) [inline client_unix_test.go]: I'd suggest enumerating platforms without unix socket support here, since this list is shorter.

**Comment by valyala** [comment]
Thanks for the contribution!

**Comment by kataras** [comment]
@ernado I already fixed it , I just don't use the ctx.SetPath, and make the redirect manually. I'm checking this repository everyday if someone post about the issue I had I will add pr a complete solution, thanks for your contribution both of you!

**Pull request merged**